src/Twig/ContentExtension.php

```diff
@@ -215,4 +216,19 @@ public function getEditLink(Content $content, bool $absolute = false): ?string
             $absolute ? UrlGeneratorInterface::ABSOLUTE_URL : UrlGeneratorInterface::ABSOLUTE_PATH
         );
     }
+
+    public function getTaxonomies(Content $content): Collection
```

Comment:
this can be moved to Content entity, as something like getTaxonomiesTree (I don't have any better name in mind; this could be useful for API as well)

Reply:
I'll think about that over dinner. ;-)

Reply:
I tried moving this to `Content`, but I really didn't like it. I'd prefer to keep it here. The PHP code looked convoluted, and this nice expressive twig would have to change: `{% for type, taxonomies in record|taxonomies %}`. I remember you were very much in favor of getting everything like `link` and `excerpt` out of `Content` and as a filter, so I think this should be as well.